light-clients/ics10-grandpa/build.rs

```diff
@@ -17,6 +17,6 @@ use std::io::Result;
 
 fn main() -> Result<()> {
 	// compile our proto files
-	prost_build::compile_protos(&["src/proto/grandpa.proto"], &["src/"])?;
+	prost_build::compile_protos(&["src/proto/grandpa.proto", "--experimental_allow_proto3_optional"], &["src/"])?;
```

Comment:
I'm actually unsure of whether this works properly. Otherwise check https://github.com/ComposableFi/centauri/blob/master/light-clients/ics10-grandpa/build.rs#L21